### DISCUSSION TITLE
Automatic Log Clearing Configuration

### DISCUSSION THREAD

**Discussion by vildanbina** [laradumps/app#330] [Ideas]
# 🛻 LaraDumps Feature Request

## Summary

I'm interested in whether there is a feature to automatically clear logs after each request to maintain a clean development environment.

### Why is this needed?

During development, logs can quickly accumulate and clutter the logging output, making it difficult to track recent and relevant entries. Having a way to automatically clear logs after each request would help in maintaining a clean log state, improving the readability and manageability of log data.

### Suggested Solution(s)

I suggest implementing a configurable option in the LaraDumps settings that allows developers to enable or disable automatic log clearing after each request. This could be a simple boolean configuration in the LaraDumps config file.

### Related issue(s)/PR(s)

As of now, I am not aware of any related issues or pull requests. This feature request could serve as a starting point for further discussion and implementation.

**Comment by luanfreitasdev** [comment]
@vildanbina . Currently, we can add 'ds()->clear();' command at the beginning of the requests, which solves it. But I'm thinking about toggling the app to "pause" the dumps. What do you think?

**Comment by vildanbina** [comment]
@luanfreitasdev both options would be useful.

`ds()->clear()` can be used in the `AppServiceProvider`, which would clear the logs for each request. However, being able to pause it via an env variable would also be useful.

Thank you
